light_node/src/configuration.rs

```diff
@@ -683,6 +690,7 @@ fn validate_required_args(args: &clap::ArgMatches) {
         )),
     );
     validate_required_arg(args, "bootstrap-db-path", None);
+    validate_required_arg(args, "context-stats-db-path", None);
```

Comment:
@sebastiencs just rechecking, and this should be an optional argument. We don't always want to measure this things (*we* do, but regular users probably don't).

Reply:
Great, thank you, will merge in a bit.